cmd-graph.c

```diff
@@ -539,8 +704,12 @@ static int print_graph(struct uftrace_graph *graph, struct opts *opts)
 		print_backtrace(graph);
 	}
 
+	setup_field(opts);
+
 	if (graph->root.time || graph->root.nr_edges) {
 		pr_out("========== FUNCTION CALL GRAPH ==========\n");
+		if (opts->fields != NULL && strcmp(opts->fields, "none"))
```

Comment:
I think it'd be better using `if (!list_empty(&output_fields))`.

Reply:
I agree that, otherwise the output can be inconsistent like below:
```patch
$ diff <(uftrace graph) <(uftrace graph -f time)
--- /dev/fd/63	2017-12-16 06:53:20.246154501 +0900
+++ /dev/fd/62	2017-12-16 06:53:20.238154395 +0900
@@ -1,5 +1,6 @@
 # Function Call Graph for 't-abc' (session: 368d2cdf6c16a307)
 ========== FUNCTION CALL GRAPH ==========
+#   TIME      FUNCTION
    2.882 us : (1) t-abc
    0.685 us :  +-(1) __monstartup
             :  |
```